### ISSUE TITLE
The value of 'loss' is fluctuating when train to the third epoch, And it no longer decrease

### ISSUE THREAD

**Issue by quxu91** [megvii-model/MOTR#40]
Thanks for your wonderful jobs!
I try to train on the DanceTrack to realize the result as the paper, but when it runs to the third epoch, the value of 'loss' is fluctuating and it no longer decrease . I want to know if it is necessary continue to training. Can i decrease the learning rate right now.

**Comment by Soulmate7** [comment]
Hi, I get the same problem as you mentioned, and my loss is about 6 which is fluctuating between 5 and 7 ranges after several epochs and it also no longer decrease. By the way, I trained it on a 3090 with 24GB, does it make difference?

Could you tell me whether decreasing the lr is helpful or your solution If you solve the problem. Thanks!
